bundle/src/lib/header-bidding/prebid/prebid.ts

```diff
@@ -360,7 +364,7 @@ const initialise = (window: Window, consentState: ConsentState): void => {
 	];
 
 	if (getConsentFor('id5', consentState)) {
-		userIds.push({
+		const userIdsForId5 = {
```

Comment:
_optional change_ as this refers to one user ID configuration (singular) not multiple 

```suggestion
		const id5UserId = {
```